javascript/ql/src/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -478,6 +478,28 @@ module NodeJSLib {
               DataFlow::moduleImport("util-promisifyall")
             ].getACall()
         )
+        or
+        // const fs = require('fs');
+        // module.exports = methods.reduce((obj, method) => {
```

Comment:
```suggestion
        // let fs_copy = methods.reduce((obj, method) => {
```
We do not use `module.exports = ` in the implementation, yet it is a special enough construct that I expected to find a reference to it in the implementation when it is used in the example.